styleguide/_themes/global/scss/forms.scss

```diff
@@ -114,3 +114,51 @@ table {
 .table > thead > tr > th {
   font-weight: unset;
 }
+
+.formModal-closeIcon {
+  opacity: .7;
+}
+
+.formModal-title {
+  text-align: center;
+}
+
+.formModal-content {
+  border-radius: 2px;
+  padding: 15px;
+}
+
+.formModal-header {
+  border-bottom: 0;
+}
+
+.formModal-body {
+  padding: 10px 20px;
+}
+
+.formModal-dialog {
+
+  @media (min-width: $screen-sm-min) {
+    width: 440px;
```

Comment:
Don't place the media query inside the selector.  Move it outside the selector instead.  

i.e.: 
```
@media (min-width: $screen-sm-min) {
  .formModal-dialog {
    width: 440px;
  }
}
```